spec/lib/blacklight/open_struct_with_hash_access_spec.rb

```diff
@@ -62,15 +62,6 @@
     end
   end
 
-  describe "#replace" do
```

Comment:
I'm confused; are we just dropping these methods without deprecation?

Reply:
Ostruct in Ruby 3 has quite a different implementation than in Ruby 2.  As such, I'm unable to devise a method such that these continue to work in the way they did in ruby 2.  Any ideas?